demo/demo.js

```diff
@@ -53,7 +63,10 @@ export function init(formDefaults, render) {
 		storeParams(formDefaults);
 	});
 
-	setTimeout(renderInternal, 1000);
+	// if form has valid values on page load then render
+	if (formValues.baseUrl && (formValues.webcastId || formValues.videoId)) {
+		setTimeout(renderInternal, 1000);
+	}
```

Comment:
It is called every time. This line just refers to the initial page/init load. Otherwise you get an unnecessary uncaught error "baseUrl is not defined".